LabBuilder/LabBuilder.psm1

```diff
@@ -1414,8 +1414,8 @@ function Get-LabVMTemplates {
                 processorcount = $Template.ProcessorCount;
                 exposevirtualizationextensions = if ($Template.ExposeVirtualizationExtensions)
                     {
-                        $Template.ExposeVirtualizationExtensions
-                    }
+						exposevirtualizationextensions=$Template.ExposeVirtualizationExtensions
```

Comment:
The ```exposevirtualizationextensions=``` shouldn't be needed I think. These lines of code can be thought of as behaving like a ternary operator (I do wish PS had a more compact ternary operator).